### DISCUSSION TITLE
Add ability to see mod settings all at once

### DISCUSSION THREAD

**Discussion by Moltanicaa** [ppy/osu#13162] [Ideas]
**Describe the new feature:** 
I can enable multiple mods with custom values, but on the details screen, I can only show one mod (which I hover over). It would be convenient to add the ability to either display all mods at once (by hovering over any of them), or add information about the map speed to the bottom of the results screen.
![osu_2020-10-07_19-54-13](https://user-images.githubusercontent.com/59089206/95363108-aee27680-08d7-11eb-89eb-a2af79b97b43.jpg)
![osu_2020-10-07_19-54-57](https://user-images.githubusercontent.com/59089206/95363116-b0ac3a00-08d7-11eb-87aa-892f8957c302.jpg)

**Comment by peppy** [comment]
not sure about this, but mod icons do really need a way to show they have settings adjusted from the defaults (a little "light" circle in their corner or something).

if we decide to show all the modifications at once, it may be best done in tabulated form in the extended results area.
